bookstore/book/templates/base.html

```diff
@@ -30,7 +30,7 @@
 
             <div class="dropdown-menu" aria-labelledby="navbarDropdown">
                 {% for category in categories %}
-                <a class="dropdown-item" href="{% url 'book:category_display' category.catid %}">{{ category.cat_name
+                <a id="dropdown-item" class="dropdown-item" href="{% url 'book:category_display' category.catid %}">{{ category.cat_name
```

Comment:
This need a fix, dropwdown list displayes '{{ category.cat_name }}' literaly. Added issue #31.